hyperspy/io_plugins/msa.py

```diff
@@ -189,8 +192,9 @@ def file_reader(filename, encoding='latin-1', **kwds):
                     parameters[parameter] = keywords[clean_par]['dtype'](
                         value.replace(' ', ''))
                 except:
-                    print("The %s keyword value, %s " % (parameter, value) +
-                          "could not be converted to the right type")
+                    warnings.warn(
+                        "The %s keyword value, %s " % (parameter, value) +
+                        "could not be converted to the right type")
```

Comment:
probably here too - I suspect it would only warn the user when opening the first file that raises the warning